packages/capabilities/test/capabilities/access.test.js

```diff
@@ -371,3 +371,256 @@ describe('access capabilities', function () {
     })
   })
 })
+
+describe('access/delegate', () => {
+  it('can create valid delegations and authorize them', async () => {
+    const issuer = alice
+    const audience = service.withDID('did:web:web3.storage')
+    const bobCanStoreAllWithAlice = await delegate({
+      issuer: alice,
+      audience: bob,
+      capabilities: [{ can: 'store/*', with: alice.did() }],
+    })
+    const validInvocations = [
```

Comment:
This is really hard to follow and if one of the cases fails the whole test will fail as well. Could you generate tests instead meaning have loop that calls `it` ? It would be easier to follow and failure in one case will not fail the whole suite.

Reply:
https://github.com/web3-storage/w3protocol/pull/420/commits/edeee6b9d3d619c8468b93c87d1276255effc68f